web/modules/custom/owner/owner.behat.inc

```diff
@@ -33,37 +33,81 @@ class OwnerSubContext extends DrupalSubContextBase implements DrupalSubContextIn
    * Table format:
    * | name | Mightily Oats |
    *
-   * @param TableNode $collection_table
-   *   The collection data.
+   * @param TableNode $person_table
+   *   The person table.
    *
    * @Given (the following )person:
    */
-  public function givenPerson(TableNode $collection_table) {
+  public function givenPerson(TableNode $person_table) {
     // Convert aliased field names.
-    $values = $this->convertFieldAliases('person', $collection_table);
+    $values = $this->convertFieldAliases('person', $person_table->getRowsHash());
 
     $this->createOwner('person', $values);
   }
 
   /**
+   * Creates a number of person owners with data provided in a table.
+   *
+   * Table format:
+   * | name          |
+   * | Mightily Oats |
+   * | John Irwin    |
+   *
+   * @param TableNode $person_table
+   *   The person table.
+   *
+   * @Given (the following )persons:
```

Comment:
Even if the entity is called person and the plural is person entities, I wouldn't go against the dictionary just to sound BDD. I would either set people, or person entities. An entity after all is something beyond programming that declares...well..an entity. I don't think it's that bad to use it just because it's heavily used in Drupal.